moler/cmd/unix/echo.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+"""
+Echo command module.
+"""
+__author__ = 'Agnieszka Bylica'
+__copyright__ = 'Copyright (C) 2018, Nokia'
+__email__ = 'agnieszka.bylica@nokia.com'
+
+
+from moler.cmd.unix.genericunix import GenericUnixCommand
+from moler.exceptions import ParsingDone
+
+
+class Echo(GenericUnixCommand):
+    def __init__(self, connection, options=None, text=None, output_file=None, prompt=None, new_line_chars='\n'):
+        super(Echo, self).__init__(connection=connection, prompt=prompt, new_line_chars=new_line_chars)
+
+        self.options = options
+        self.text = text
+        self.output_file = output_file
+
+        self.current_ret['RESULT'] = list()
+
+    def build_command_string(self):
+        cmd = "echo"
+        if self.options:
+            cmd = "{} {}".format(cmd, self.options)
+        if self.text:
+            cmd = "{} {!r}".format(cmd, self.text)
+        if self.output_file:
+            cmd = "{} > {}".format(cmd, self.output_file)
```

Comment:
When `output_file` parameter is passed into command there is only one mode to write data into file: > which replace all lines in files with new data.
There should be another one write mode: >> which mean append data to existing lines in specific file. 
Please add write_mode parameter into command.

Reply:
Can write_mode have a default value ">'?

Reply:
I think yes.